product-backend/src/main/java/org/kainos/ea/db/CapabilityDao.java

```diff
@@ -0,0 +1,61 @@
+package org.kainos.ea.db;
+
+
+import org.kainos.ea.model.Capability;
+import org.kainos.ea.model.CapabilityRequest;
+import org.kainos.ea.model.JobRole;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class CapabilityDao {
+
+
+    public Optional<List<Capability>> getEveryCapabilityLead(Connection c) throws SQLException {
```

Comment:
You can leave that optional here
```suggestion
    public List<Capability> getEveryCapabilityLead(Connection c) throws SQLException {
```